src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -50,42 +50,38 @@ class MoreInfoMediaPlayer extends LitElement {
     const controls = computeMediaControls(stateObj);
 
     return html`
-      ${!controls
-        ? ""
-        : html`
-            <div class="controls">
-              <div class="basic-controls">
-                ${controls!.map(
-                  (control) => html`
-                    <ha-icon-button
-                      action=${control.action}
-                      @click=${this._handleClick}
-                      .path=${control.icon}
-                      .label=${this.hass.localize(
-                        `ui.card.media_player.${control.action}`
-                      )}
-                    >
-                    </ha-icon-button>
-                  `
+      <div class="controls">
```

Comment:
It doesn't matter if these divs are empty? No padding/margin whatever?

Reply:
nope.